lib/model/unittest/CEventRateModelTest.cc

```diff
@@ -44,218 +44,163 @@
 
 #include <stdint.h>
 
-BOOST_TEST_DONT_PRINT_LOG_VALUE(TStrVec::iterator)
+BOOST_TEST_DONT_PRINT_LOG_VALUE(CModelTestFixtureBase::TStrVec::iterator)
 
 BOOST_AUTO_TEST_SUITE(CEventRateModelTest)
 
 using namespace ml;
 using namespace model;
 
 namespace {
-
 const std::string EMPTY_STRING;
```

Comment:
This is in an unnamed namespace in several of the files.  Could it be made a public or protected static constant on the test fixture?  Then it would still be usable in the tests without qualification, but wouldn't be duplicated.